src/processes.c

```diff
@@ -1295,25 +1320,35 @@ static int read_fork_rate (void)
 		int fields_num;
 
 		fields_num = strsplit (buffer, fields,
-				STATIC_ARRAY_SIZE (fields));
-		if (fields_num != 2)
-			continue;
+			STATIC_ARRAY_SIZE (fields));
 
-		if (strcmp ("processes", fields[0]) != 0)
-			continue;
+		if((fields_num == 2) && (strcmp ("processes", fields[0]) == 0))
+		{
+			status = parse_value (fields[1], &fr_value,
+				DS_TYPE_DERIVE);
 
-		status = parse_value (fields[1], &value, DS_TYPE_DERIVE);
-		if (status == 0)
-			value_valid = 1;
+			if (status == 0)
+				fr_value_valid = 1;
+		}
+		else if ((fields_num >= 2) && (strcmp ("intr", fields[0]) == 0))
+		{
+			status = parse_value (fields[1], &intr_value,
+				DS_TYPE_DERIVE);
+
+			if (status == 0)
+				intr_value_valid = 1;
+		}
 
-		break;
+		if(fr_value_valid && intr_value_valid)
+			break;
 	}
 	fclose(proc_stat);
 
-	if (!value_valid)
+	if (!fr_value_valid || !intr_value_valid)
 		return (-1);
 
-	ps_submit_fork_rate (value.derive);
+	ps_submit_fork_rate (fr_value.derive);
+	ps_submit_syscalls (fr_value.derive + intr_value.derive);
```

Comment:
Can you please add a comment why these two seemingly unrelated metrics need to be added together here?